doc/rifle.1

```diff
@@ -1,4 +1,4 @@
-.\" Automatically generated by Pod::Man 4.10 (Pod::Simple 3.35)
+.\" Automatically generated by Pod::Man 4.11 (Pod::Simple 3.35)
```

Comment:
Remove this file from your commits please. We only want it committed *if* it's changed.